javascript/javascript1/week2/home-work/event.js

```diff
@@ -0,0 +1,14 @@
+"use strict"
+
+
+// Event Application
+
+ 
+ 
+function getEventWeekday(days) {
```

Comment:
Good attempt, although I don't think this does exactly what the exersize asked. I would check out victoria's solution for inspiration: https://github.com/panivita/hyf-homework/blob/javascript/javascript1/week2/javascript/javascript1/week2/homework/homework.js